aliroot.sh

```diff
@@ -48,6 +48,7 @@ cmake $SOURCEDIR                                                     \
       ${ALIEN_RUNTIME_ROOT:+-DALIEN="$ALIEN_RUNTIME_ROOT"}           \
       ${FASTJET_ROOT:+-DFASTJET="$FASTJET_ROOT"}                     \
       ${DPMJET_ROOT:+-DDPMJET="$DPMJET_ROOT"}                        \
+      ${ZEROMQ_ROOT:+-DZEROMQ=$ZEROMQ_ROOT}                          \
```

Comment:
This one will force people to rebuild AliRoot. We usually allow that only when we do a new release. Can you split this PR in two and do this change in dev, so that @dberzano can pick up the change next time he builds AliRoot?